react/training/src/app/screens/Login/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+
+import LoginForm from './components/LoginForm/index';
+
+class Login extends Component {
+  state = {
+    isLoggedIn: false
+  };
```

Comment:
add space here. 
This is ok in this pr, but in the next one isLoggedIn should be in the store after validating with you API if it is an valid user.
So for now this is great 👍